ReduxMovieDB/Controllers/MovieDetailViewController.swift

```diff
@@ -78,7 +79,11 @@ class MovieDetailViewController: UITableViewController {
 
         return defaultHeight
     }
-
+    @IBAction func toggleFavorite(_ sender: Any) {
+        mainStore.dispatch(MainStateAction.toggleFavoriteMovie)
+        favoriteButton.setTitle(mainStore.state.isCurrentFavorite ? "★" : "☆", for: .normal)
+    }
```

Comment:
This should be done with RxSwift in didSet (see the list controller for examples)

Reply:
Also, when you toggle and go back to the other screen, the movie is updated with a star.

Reply:
Got it 👌